nncf/common/pruning/shape_pruning_processor.py

```diff
@@ -126,8 +126,6 @@ def prune_cluster_shapes(self,
 
         for node in cluster.elements:
             output_channels[node.node_name] -= pruned_elems
-            if node.is_depthwise:
-                input_channels[node.node_name] -= pruned_elems
```

Comment:
The reason to remove this part is that depthwise nodes are in `next_nodes_info`. (Yi)

Reply:
Yes, this part here is important and will brake shape pruning in the filter pruning algo